reana_server/reana_admin.py

```diff
@@ -507,8 +508,9 @@ def list_quota_usage(
 @click.pass_context
 def list_quota_resources(ctx, admin_access_token):
     """List quota resources."""
-    resource_names = [resource.name for resource in Resource.query]
-    click.echo("\n".join(resource_names))
+    click.echo("Available resources are:")
+    for resource in Resource.query:
+        click.echo(f"{resource.type_.name} ({resource.name})")
```

Comment:
Shouldn't it be the other way around? First the resource name and then resource type in parenthesis. In the end, the unique value here is the resource name.

Reply:
No, the idea is that on one-type-one-resource scenario one can only use the type for specifying the limits, etc. So it makes sense to have the type of a resource as a main value

Reply:
For me this:
```
fast processing time (cpu)                                                                                                                                                                                                                                                               
shared storage (disk)                                                                                                                                                                                                                                                        
slow processing time (cpu)
```

looks better than this:

```
cpu (fast processing time)
disk (shared storage)                                                                                                                                                                                                                                               
cpu (slow processing time)
```

Furthermore, the command is called `quota-resources`, no `quota-resource-types`!

Reply:
Our primary use case right now is one-type-one-resource scenario, so second option for me makes more sense. We refer to the resource type as a `resource` in client and admin facing commands:
- reana-client quota-show --resource disk -h
- reana-admin quota-set --resource disk ...